src/Net.cpp

```diff
@@ -181,3 +181,16 @@ bool Ban::operator ==(const Ban &other) const {
 bool Ban::isValid() const {
 	return haAddress.isValid() && (iMask >= 8) && (iMask <= 128);
 }
+
+QString Ban::toString() const {
+	return QString("Hash: \"%1\", Host: \"%2\", Mask: \"%3\", Username: \"%4\", Reason: \"%5\", BanStart: \"%6\", BanEnd: \"%7\" %8").arg(
+		qsHash,
+		haAddress.toString(),
+		haAddress.isV6() ? QString::number(iMask) : QString::number(iMask-96),
+		qsUsername,
+		qsReason,
+		qdtStart.toLocalTime().toString("yyyy-MM-dd hh:mm:ss"),
```

Comment:
Can we please add a TZ to the date string? Just to avoid confusion? People might think it's UTC. (What do our log timestamps use, actually?)